src/api/app/components/add_review_component.html.haml

```diff
@@ -0,0 +1,19 @@
+.dropdown#add-review-component
+  .btn.btn-success.dropdown-toggle{ type: 'button', data: { toggle: 'dropdown'}, aria: { expanded: 'false' } }
+    Review
+  .dropdown-menu.dropdown-menu-right
+    - @my_open_reviews.each_with_index do |review, index|
```

Comment:
If there is only one, why show the type?

Reply:
true, let me change this